jenkins/scripts/VersionSelectorScript.groovy

```diff
@@ -61,8 +61,11 @@ def buildExclusions = [
   [ /sharedlibs_withoutssl/,          anyType,     lt(10)  ],
   [ /sharedlibs_shared/,              anyType,     lt(9)   ],
 
-  // -------------------------------------------------------
+  // OSX ---------------------------------------------------
+  [ /^osx1010/,                       anyType,     gte(11) ],
```

Comment:
Can it be `gte(10)`? There seems to be consensus in https://github.com/nodejs/node/pull/21883 that dropping support for 10.10 is not semver-major.

Reply:
Although it's not semver major, we tend not to "pull the rug under the feet" of current users without an actual need.
I'm +1 on landing as is, and re-evaluating if the need arises.

Reply:
(this is assuming the bump to V8 6.8 is not planned for v10)

Reply:
I'm now testing backward compatibility of binaries built with 10.11 on 10.10. If the tests pass, I'll be 100% with excluding 10.10 for node v10 as well.